selftests/functional/test_basic.py

```diff
@@ -350,6 +350,9 @@ def test_runner_timeout(self):
         # Ensure no test aborted error messages show up
         self.assertNotIn("TestAbortedError: Test aborted unexpectedly", output)
 
+    @unittest.skipIf(os.environ.get("AVOCADO_CHECK_FULL") != "1",
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
I'd actually prefer skip message that would reflect the actual test's reason. Here it's that this test is sensitive for timing and is skipped unless AVOCADO_CHECK_FULL=1. Again, it's just a suggestion as both seems fine by me.